# Compare replication to Bessen & Hunt (2007)

There is only one differently classified patent among the corpus of 399 patents from Bessen & Hunt (2007). Let us look at this specific patent.

In [22]:
import sys
sys.path.append('../..')

import pandas as pd
import numpy as np

from bld.project_paths import project_paths_join as ppj

In [23]:
df = pd.read_pickle(ppj('OUT_ANALYSIS', 'replication_bh.pkl'))

In [24]:
df.iloc[9, np.where(df.iloc[9])[0]]

CLASSIFICATION_ALGORITHM          Software
CLASSIFICATION_MANUAL         Non-Software
PATENTNR                           5489660
DESCRIPTION_SOFTWARE                  True
DESCRIPTION_COMPUTER                  True
DESCRIPTION_PROGRAM                   True
DESCRIPTION_CHROMATOGRAPHY            True
CLASSIFICATION_REPLICATION    Non-Software
Name: 9, dtype: object

The patent in question is no. 5489660. As one can see it has all the necessary ingredients to be classified as a software patent. But chromatography is also mentioned in the text and given the author's algorithm, it must not be mentioned in the description or the abstract. Probably, the word was overlooked while reading the patent text.